File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Rain.java
package com.riders.thelab.data.remote.dto.weather;

import com.google.gson.annotations.SerializedName;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
public class Rain {

    // Rain volume for the last 1 hour, mm
    @SerializedName("1h")
    double lastHour;

    // Rain volume for the last 3 hour, mm
    @SerializedName("3h")
    double lastThreeHour;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/weather/Rain.kt
package com.riders.thelab.data.remote.dto.weather

import kotlinx.serialization.SerialName
import java.io.Serializable

@kotlinx.serialization.Serializable
data class Rain(
    // Rain volume for the last 1 hour, mm
    @SerialName("1h")
    val lastHour: Double = 0.0,

    // Rain volume for the last 3 hour, mm
    @SerialName("3h")
    val lastThreeHour: Double = 0.0
) : Serializable
